nvtabular/ops/data_stats.py

```diff
@@ -13,95 +13,290 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import base64
+import logging
+import os
+
+import dask.array as da
 import dask.dataframe as dd
 import numpy as np
 
 from nvtabular.dispatch import DataFrameType, annotate
 
 from ..column import Columns
 from .base import Operator
-from .moments import _custom_moments
+from .data_schema import DataSchema
 from .stat_operator import StatOperator
 
+LOG = logging.getLogger("nvtabular")
 
-class DataStats(StatOperator):
-    def __init__(self):
-        super().__init__()
+
+class DataStats(DataSchema):
```

Comment:
This seems like an unusual inheritance chain, at least from the class names. I normally wouldn't expect stats to be a schema.